elide-core/src/main/java/com/yahoo/elide/core/PersistentResource.java

```
@@ -819,13 +819,14 @@ public PersistentResource getRelation(String relation, String id) {
         /* If this is a bulk edit request and the ID we are fetching for is newly created... */
         if (requestScope.isMutatingMultipleEntities()
                         && requestScope.getObjectById(dictionary.getJsonAliasFor(entityType), id) != null) {
-            filterExpression = Optional.empty();
-            // NOTE: We can safely _skip_ tests here since we are only skipping READ checks on
-            // NEWLY created objects. We assume a user can READ their object in the midst of creation.
-            // Imposing a constraint to the contrary-- at this moment-- seems arbitrary and does not
-            // reflect reality (i.e. if a user is creating an object with values, he/she knows those values
-            // already).
-            skipNew = true;
+            if (!this.checkRelation(relation)) {
+                throw new ForbiddenAccessException("oops");
+            }
+            Object entity = requestScope.getObjectById(dictionary.getJsonAliasFor(entityType), id);
+            PersistentResource resource = requestScope.getNewPersistentResources().stream()
+                    .filter(r -> r.getObject() == entity).findFirst().orElse(null);
```

orElse we should throw an exception.